kokoro/macos/build.sh

```diff
@@ -29,6 +29,21 @@ unzip -q android-ndk-r21-darwin-x86_64.zip -d android
 export ANDROID_HOME=$PWD/android
 export ANDROID_NDK_HOME=$PWD/android/android-ndk-r21
 
+# Get the JDK and JRE from our mirror. This needs to be after the Android updates above.
+JDK_BUILD=zulu11.39.15-ca
+JDK_VERSION=11.0.7
+JDK_NAME=$JDK_BUILD-jdk$JDK_VERSION-macosx_x64
+JRE_NAME=$JDK_BUILD-jre$JDK_VERSION-macosx_x64
+curl -L -k -O -s https://storage.googleapis.com/jdk-mirror/$JDK_BUILD/$JDK_NAME.zip
+echo "e5ea71dd86eefe6e2ef78720ea729f26a8b5c279bcb2f1770745698ef374f9b8 $JDK_NAME.zip" | sha256sum --check
```

Comment:
I get an error here on macOS since i don't have sha256sum. Instead we should use 
```
echo "e5ea71dd86eefe6e2ef78720ea729f26a8b5c279bcb2f1770745698ef374f9b8 $JDK_NAME.zip" | shasum -c
```